<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>3</number>
          </property>
          <property name="leftMargin" >
           <number>3</number>
          </property>
          <property name="topMargin" >
           <number>3</number>
          </property>
          <property name="rightMargin" >
           <number>3</number>
          </property>
          <property name="bottomMargin" >
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_open" >
            <property name="text" >
             <string>Open report file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox" >
            <property name="title" >
             <string>Class</string>
            </property>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>3</number>
             </property>
             <property name="leftMargin" >
              <number>3</number>
             </property>
             <property name="topMargin" >
              <number>3</number>
             </property>
             <property name="rightMargin" >
              <number>3</number>
             </property>
             <property name="bottomMargin" >
              <number>3</number>
             </property>
             <item>
              <widget class="QComboBox" name="comboBox_class" />
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_selectAllPupils" >
            <property name="text" >
             <string>Select all pupils</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_unselectAllPupils" >
            <property name="text" >
             <string>Unselect all pupils</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>71</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_shrink" >
         <property name="acceptDrops" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Shrink to A4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_landscape" >
         <property name="text" >
          <string>Landscape orientation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_reverse" >
         <property name="text" >
          <string>Last page first</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_pdf" >
         <property name="text" >
          <string>Print to file (pdf)</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Pupils</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="leftMargin" >
         <number>3</number>
        </property>
        <property name="topMargin" >
         <number>3</number>
        </property>
        <property name="rightMargin" >
         <number>3</number>
        </property>
        <property name="bottomMargin" >
         <number>3</number>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget_pupils" >
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Sheet selection</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="leftMargin" >
         <number>3</number>
        </property>
        <property name="topMargin" >
         <number>3</number>
        </property>
        <property name="rightMargin" >
         <number>3</number>
        </property>
        <property name="bottomMargin" >
         <number>3</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButton_pagesM" >
          <property name="text" >
           <string>A3 sheets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_pagesS" >
          <property name="text" >
           <string>A4 sheets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_pagesI" >
          <property name="text" >
           <string>Individual pages</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_pages" />
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_allPages" >
          <property name="text" >
           <string>Select all</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Messages</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>3</number>
      </property>
      <property name="leftMargin" >
       <number>3</number>
      </property>
      <property name="topMargin" >
       <number>3</number>
      </property>
      <property name="rightMargin" >
       <number>3</number>
      </property>
      <property name="bottomMargin" >
       <number>3</number>
      </property>
      <item>
       <widget class="QTextEdit" name="textEdit" >
        <property name="lineWrapMode" >
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_preview" >
       <property name="text" >
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_print" >
       <property name="text" >
        <string>Print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>583</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel" >
     <x>430</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_pagesM</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_shrink</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>500</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>146</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_pagesM</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_landscape</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>544</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
